src/core/file_sys/fs.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+#include <mutex>
+#include <string>
+#include <vector>
+
+#include "common/fs_file.h"
+
+namespace Core::FileSys {
+
+class MntPoints {
+  public:
+    struct MntPair {
+        std::string host_path;
+        std::string guest_path;  // e.g /app0/
+    };
+
+    MntPoints() {}
+    virtual ~MntPoints() {}
+    void mount(const std::string& host_folder, const std::string& guest_folder);
```

Comment:
Maybe pass std::filesystem::path directly to mount/unmount, so the API will expose paths. If needed the class can convert and store the strings interally